### ISSUE TITLE
Set up C++ runtime tests on Windows

### ISSUE THREAD

**Issue by KvanTTT** [antlr/antlr4#3389]
And include them to AppVeyor

**Comment by parrt** [comment]
maybe @mike-lischke has ideas.

**Comment by mike-lischke** [comment]
The build is indeed simple. There's that `runtime/Cpp/deploy-windows.cmd` which can serve as template.

**Comment by KvanTTT** [comment]
Yes, I've tried pipeline manually and it eventually works. Now I'm going to translate it into Java code. Actually, no bat file is required at all.

**Comment by mike-lischke** [comment]
@KvanTTT You may want to wait for the next C++ patch #3398 from @jcking who remove the external utf8lib reference. This way the build should again become straight forward.

**Comment by KvanTTT** [comment]
Ok.
